templates/workflows/kotlin_postgres_integration_tests.yml

```diff
@@ -1,92 +0,0 @@
-name: integration test template for kotlin
```

Comment:
rest of the templates were removed, but this was left for some reason, so removing too